tool/gravity/cli/config.go

```diff
@@ -383,6 +387,11 @@ func (i *InstallConfig) NewInstallerConfig(
 
 }
 
+func (i *InstallConfig) validateApplicationDir() error {
```

Comment:
This is an additional test in attempt to move as much validation as possible to as early as possible.
The issue here is that the way client/service interaction works - client needs to connect to the service in order to display anything (incl. errors). If the service has an initialization time failure, it is currently difficult to catch and respond to. I'm still considering options as to how best solve this.